packages/components/src/VirtualizedList/RendererSelector.test.js

```diff
@@ -93,4 +97,100 @@ describe('RendererSelector', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 		expect(wrapper.getElement().props.rowRenderer.displayName).toBe('VirtualizedList(RowLarge)');
 	});
+
+	it('should render the grid with the default NoRows', () => {
+		// when
+		const wrapper = shallow(
+			<RendererSelector
+				collection={collection}
+				height={600}
+				id={'my-list-id'}
+				isActive={jest.fn()}
+				isSelected={jest.fn()}
+				onRowClick={jest.fn()}
+				rowHeight={50}
+				selectionToggle={jest.fn()}
+				type={LARGE}
+				width={1024}
+			>
+				{contentFields}
+			</RendererSelector>,
+		);
+
+		// then
+		expect(wrapper.props().noRowsRenderer()).toMatchSnapshot();
+	});
+
+	it('should render the grid with the default NoRows', () => {
+		// when
+		const wrapper = shallow(
+			<RendererSelector
+				collection={collection}
+				height={600}
+				id={'my-list-id'}
+				isActive={jest.fn()}
+				isSelected={jest.fn()}
+				onRowClick={jest.fn()}
+				rowHeight={50}
+				selectionToggle={jest.fn()}
+				type={LARGE}
+				width={1024}
+			>
+				{contentFields}
+			</RendererSelector>,
+		);
+
+		// then
+		expect(wrapper.props().noRowsRenderer()).toMatchSnapshot();
+	});
+
+	it('should render the table with the noRowsRenderer', () => {
+		// when
+		const wrapper = shallow(
+			<RendererSelector
+				collection={collection}
+				noRowsRenderer={NoRowsRenderer}
+				height={600}
+				id={'my-list-id'}
+				isActive={jest.fn()}
+				isSelected={jest.fn()}
+				onRowClick={jest.fn()}
+				selectionToggle={jest.fn()}
+				sort={jest.fn()}
+				sortBy={'name'}
+				sortDirection={'DESC'}
+				width={1024}
+			>
+				{contentFields}
+			</RendererSelector>,
+		);
+
+		// then
+		expect(wrapper.props().noRowsRenderer()).toMatchSnapshot();
+	});
+
+	it('should render the table with the noRowsRenderer', () => {
```

Comment:
Same test than previous.
You meant to mix custom/default with table/large, but you ended up with table-custom x2 and large-default x2 
:)